openapi/openapi.yaml

```
@@ -362,6 +362,15 @@ tags:
   - name: Usage
     description: |-
       Reports the product usage of a subscription item for metered billing purposes.
+  - name: Quotes
+    description: |-
+      A quote is an organization's suggestion to purchase items. It can
+      contain subscription and one-time sale items. When a quote
+      contains one or more subscription items, it is a subscription
+      order quote.
+
+      A quote contains an initial invoice preview and can be accepted to
+      become an actual order.
```

suggestion: suggestion to use the same style as the other descriptions in `openapi.yaml`
```suggestion
      Use quote operations to create and manage quotations.
      Quotations describe the cost of goods or services to potential customers before they commit to a purchase.
      A quote contains an initial invoice preview that can be accepted to become an order.
      
      A quote can contain subscription and one-time sale items.
      When a quote contains one or more subscription items, it is a subscription order quote
```